engine/src/build/mac/find_sdk.py

```diff
@@ -65,7 +65,6 @@ def main():
                     action="store", type="string", dest="symlink",
                     help="Whether to create a symlink in the buildroot to the SDK.")
   (options, args) = parser.parse_args()
-  min_sdk_version = args[0]
```

Comment:
Turns out this is entirely unused.

Reply:
curious why it's no longer in use? min sdk version sounds like something useful.

Reply:
It was removed in flutter/buildroot#912. 

We now use `xcrun --sdk macosx --show-sdk-path` to get the SDK path rather than try to work it out from a list of available SDKs.